crates/nu-command/src/misc/panic.rs

```diff
@@ -18,7 +22,7 @@ impl Command for Panic {
             // LsGlobPattern is similar to string, it won't auto-expand
             // and we use it to track if the user input is quoted.
             .optional("msg", SyntaxShape::String, "The glob pattern to use.")
```

Comment:
These lines above seem to be copy and paste mistakes?

Reply:
🤦🏻